src/main/resources/alfresco/heartbeat/heartbeat-repo-collector-context.xml

```diff
@@ -13,6 +15,8 @@
 
     <!-- HeartBeat authority data collector -->
     <bean id="authorityDataCollector" class="org.alfresco.heartbeat.AuthoritiesDataCollector" parent="hbBaseDataCollector" init-method="register">
+        <property name="cronExpression" value="0 0/2 * * * ?"/> <!-- every hour -->
+        <constructor-arg type = "java.lang.String" value = "acs.repository.usage.authorities"/>

```

Comment:
Can use the setter instead of constructor to set the collector name
<property name="collectorId" value="acs.repository.usage.authorities">